python/rapidsmpf/rapidsmpf/integrations/dask/shuffler.py

```diff
@@ -462,10 +472,15 @@ def rapidsmpf_shuffle_graph(
     if not isinstance(integration, DaskIntegration):
         raise TypeError(f"Expected DaskIntegration object, got {integration}.")
 
-    # Extract mapping between ranks and worker addresses
     worker_ranks: dict[int, str] = {
-        v: k for k, v in client.run(get_worker_rank).items()
+        v: k
```

Comment:
It would be good with a comment about the overhead of calling `client.run` in ucxx